.circleci/config.yml

```diff
@@ -137,6 +137,29 @@ jobs:
           command: make -C docs
       - save_sbt_cache
 
+  samples-tests:
+    machine: true
+    description: "Maven Samples Integreation Tests"
+    steps:
+      - checkout-and-merge-to-main
+      - setup_sbt
+      - restore_sbt_cache
+      - run: 
+          name: "Publish local maven"
+          command: | 
+            # tag the head with the hash
+            # dynver requires a version number prefix, ie: v0.0.1-
+            git tag v0.0.1-"$CIRCLE_SHA1"
+            echo 'export SDK_VERSION=0.0.1-${CIRCLE_SHA1}' >> $BASH_ENV
```

Comment:
I haven't followed what's been tried, but since this was mentioned on the call.

Instead of tagging, could it use `sbt -q "print sdk/version"` to get the version that will be published locally?

So something like:

```
echo "export SDK_VERSION=$(sbt -q "print sdk/version")" >> $BASH_ENV
```

Reply:
That's similar to what I was doing, but my version were more hacky. 

I was even printing the SDK_VERSION surrounded by '' to make sure there were nothing wrong with it. 

Still, when maven was fetching it, it was pulling something like `%1B%5B0m%%1B%5B0m%0.7.0-beta.10-6-1cd5e2ed-SNAPSHOT%1B%5B0m%`

Anyway, it can be related to the way I was trying to pick the version. I will give it another try.

Reply:
it works with 

```shell
echo 'export SDK_VERSION='`sbt "print sdk/version" | tail -n 2 | head -n 1` >> $BASH_ENV
```
The output without `print` was probably adding some invisible chars.